### PR TITLE
version 3.12 changelog & manpages

### PR DESCRIPTION
**Pull request by presidento**
Update news file, documentation.

### CODE REVIEW

**Comment by lbudai** [comment]
@presidento: drop the rabbitmq submodule patch

**Comment by presidento** [comment]
It was done immediately. I am finalizing it now.

**Comment by presidento** [comment]
@lbudai it is ready to be reviewed now.

File: NEWS.md [hunk 1]
```diff
@@ -1,82 +1,56 @@
...
-
- * Improve source suspended/resumed debug messages.
-
- * Rename syslog-debun to syslog-ng-debun.
-
- * Update manpages to v3.11
-
- * Remove tgz2build directory.
+* Improve syslog-ng debun functionality (#1633, #1641, #1663)
+* Java: allow to set JVM options form global syslog-ng options (#1639)
```
Comment (lbudai) [inline NEWS.md:41]: typo: from <-> form

Comment (lbudai) [reply NEWS.md:41]: nevermind, I'll fix it, there are some missing PRs, too :)

**Pull request merged**